### PR TITLE
Add random incident option

### PR DESCRIPTION
**Pull request by lmcnulty**
Resolves https://github.com/responsible-ai-collaborative/aiid/issues/1577.

Adds a url, `/random/` which redirects to the citation page for a randomly-selected incident. I imagine this being linked in the sidebar – it needs to be set in Prismic.

### CODE REVIEW

File: site/gatsby-site/i18n/locales/ja/translation.json [hunk 1]
```diff
@@ -528,5 +528,7 @@
   "Fewer filters": "フィルタを減らす",
   "Tags": "タグ",
   "Language": "言語",
-  "Read the Source": "情報源を読む"
+  "Read the Source": "情報源を読む",
+  "Redirecting...": "Redirecting...",
+  "Random Incident": "Random Incident"
```
Comment (clari182) [inline site/gatsby-site/i18n/locales/ja/translation.json:533]: Please add Japanese translations

**Review by clari182 — CHANGES_REQUESTED** [review]
Looks good! Just 2 japanese translations missing, and can you add some tests for this?

**Comment by kepae** [comment]
Does this PR build? Errors on the test build pipeline: https://github.com/responsible-ai-collaborative/aiid/actions/runs/8929044728/job/24525956231?pr=2767

File: site/gatsby-site/i18n/locales/ja/translation.json (hunk 1 above)
Comment (lmcnulty) [reply site/gatsby-site/i18n/locales/ja/translation.json]: Attempted. For "redirecting..." I went with "リダイレクト中..." which got from Google translate and [seems to be used for this exact purpose](https://www.google.com/search?q=%22%E3%83%AA%E3%83%80%E3%82%A4%E3%83%AC%E3%82%AF%E3%83%88%E4%B8%AD...%22) in a lot of sites. Then for the random button I went with "おまかせ表示" which is taken verbatim from the random page link of Japanese Wikipedia.

**Review by clari182 — COMMENTED** [review]
This looks good!! Works as expected.
I've added the new item to the sidebar on staging and prod, when this PR is merged it will be visible on the sidebar in staging (and then prod).
Do you think it would be much work to add a button in the cite page to load another random incident?
If you know you're coming from a random incident, maybe the button would say something like "Display another random incident" (or maybe something shorter and better), so it re-triggers the `/random/` path

**Comment by lmcnulty** [comment]
@clari182 I think a good way to handle this is putting a random incident link between the next and previous links on the cite page, as is often done in webcomics. Then there's no need to determine if we came from the random page.

image

**Review by clari182 — APPROVED** [review]

**Pull request merged**